### ISSUE TITLE
Notification text is too wordy.

### ISSUE THREAD

**Issue by michaelplzno** [mesosoi/silverwaregames-io-issue-tracker#192]
Just lists of names:

3❤️ - [Name1 link], [Name2 link], [Name3 link]...

**Comment by mesosoi** [comment]
Does it look better now?

**Comment by michaelplzno** [comment]
<img width="1436" alt="image" src="https://github.com/mesosoi/silverwaregames-io-issue-tracker/assets/2665989/fa65a8d8-b9a9-4dc8-8c79-92bf5f077207">

For now, make a post "Title" the first 4 words of the post and then a ... and don't repeat the same thing twice, in this one you have two versions of the post "Title"

**Comment by mesosoi** [comment]
OK
